inference-engine/src/transformations/src/transformations/remove_filtering_boxes_by_size.cpp

```diff
@@ -105,4 +106,9 @@ void ngraph::pass::RemoveFilteringBoxesBySize::remove_filtering_boxes_by_size()
 
     auto m = std::make_shared<ngraph::pattern::Matcher>(cast, "RemoveFilteringBoxesBySize");
     this->add_matcher(m, callback, PassProperty::CHANGE_DYNAMIC_STATE);
-}
\ No newline at end of file
+}
+
+bool ngraph::pass::RemoveFilteringBoxesBySize::run_on_function(std::shared_ptr<ngraph::Function> f) {
+    OV_ITT_SCOPED_TASK(ngraph::pass::itt::domains::nGraphPass_LT, "ngraph::pass::RemoveFilteringBoxesBySize");
```

Comment:
why do we act differently for matcher passes -- for some of them we measure only callback execution, for others -- the whole `run_on_function` call?

Reply:
Fixed. I removed counters from matchers and keeped in run_on_function.